0003-messaging-ipc/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: Messaging IPC
+shortname: 3/MIPC
+status: raw
+contributors: Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
```

Comment:
@rasendubi, thanks for the links. From what I've read there, editor is kind of an "owner" of the document and everybody else working on it are contributors. If my understanding is correct, @akokhanovskyi is the editor of RFC0003.